src/lib/game.js

```diff
@@ -9,7 +9,7 @@ class Game {
   }
 
   takeDamage() {
-    this.health -= this.#randomNumber();
+    this.health < this.#randomNumber() ? this.health -= this.health : this.health -= this.#randomNumber()
```

Comment:
this doesn't work - this will call randomNumber once to check whether health is less than it, then it will call it a second time and calculate the damage, meaning that you will often result in health that is not 0.

should the false be:
`this.health = 0`

Reply:
suggesting:

```
this.health -= this.#randomNumber();
this.health < 0 ? this.health = 0 : this.health